package.json

```diff
@@ -35,8 +35,8 @@
   },
```

Comment:
Please updated the version of the package.json to the next valid version

Reply:
Its already 7.4.0 and we don't have a release yet in 7.4